pywb/warcserver/access_checker.py

```diff
@@ -164,13 +164,13 @@ def check_embargo(self, url, ts):
         # embargo if newser than
         newer = self.embargo.get('newer')
         if newer:
-            actual = datetime.utcnow() - newer
+            actual = datetime.now(timezone.utc) - newer
```

Comment:
```suggestion
            actual = datetime.now(timezone.utc).replace(tzinfo=None) - newer
```

I think this is one of the few places where we do need to explicitly create a naive datetime or else an exception will be thrown due to comparison of naive and aware datetimes.

Reply:
`newer` is a relativedelta, not a datetime.

Reply:
You're absolutely right, my apologies!

Reply:
Ah that's right, the issue wasn't with `newer` but with the datetime created from a timestamp that `actual` is compared against. Fix in https://github.com/webrecorder/pywb/pull/946